AAC templates: Removing unwanted metadata (#52)
MicrosoftDocs/content-templates: templates/org/aac/aac-solution-idea.md

> <!-- 

Reviewer: ```suggestion
<!--
```

Don't add metadata to this Markdown file.
Use the browser header template to create a YAML file that contains your metadata.

> Add a brief introductory paragraph with no heading. 

Reviewer: ```suggestion
Add a brief introductory paragraph with no heading.
```

   <Long description that ends with a period.>
:::image-end:::

> <!-- 
> Under the architecture diagram, link to the Visio or PowerPoint file. 
> The link will work after the AAC team uploads your Visio or PowerPoint file to the Azure CDN. 

Reviewer: ```suggestion
<!--
Under the architecture diagram, link to the Visio or PowerPoint file.
The link will work after the AAC team uploads your Visio or PowerPoint file to the Microsoft Learn CDN.
```

### Components

<!-- 

> Add a bulleted list of all components in the architecture. 

Reviewer: ```suggestion
Add a bulleted list of all components in the architecture.
```

Other contributors:

<!--

> This section is optional. 
> List contributors and technical reviewers. 

Reviewer: ```suggestion
This section is optional.
List contributors and technical reviewers.
```
